renderer/viewer/lib/mesher/models.ts

```
@@ -454,9 +459,12 @@ function renderElement (world: World, cursor: Vec3, element: BlockElement, doAO:
 
         // TODO: correctly interpolate ao light based on pos (evaluate once for each corner of the block)
 
-        const ao = (side1Block && side2Block) ? 0 : (3 - (side1Block + side2Block + cornerBlock))
+        let ao = (side1Block && side2Block) ? 0 : (3 - (side1Block + side2Block + cornerBlock))
```

ESLint

[error] 'ao' is never reassigned. Use 'const' instead.

(prefer-const)